sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRepositoryClientLiveTests.cs

```diff
@@ -55,23 +56,85 @@ public async Task CanGetRepositoryProperties()
 
             RepositoryProperties properties = await _client.GetPropertiesAsync();
 
+            Assert.AreEqual(canList, properties.WriteableProperties.CanList);
+            Assert.AreEqual(canRead, properties.WriteableProperties.CanRead);
+            Assert.AreEqual(canWrite, properties.WriteableProperties.CanWrite);
+            Assert.AreEqual(canDelete, properties.WriteableProperties.CanDelete);
+        }
+
+        [TearDown]
+        public async Task ResetRepositoryProperties()
+        {
+            await _client.SetPropertiesAsync(
```

Comment:
Would this make sense to call in `CreateClient` as a way to ensure the client is in this state before each test? The reason I ask is that if, for some reason, you are debugging a test and cause this not to run and tests expect this state, it could get your tests into a bad state.

Reply:
You may also want to consider only calling before methods that expect this state. The main reason being that it will bloat your recordings if it is not relevant to most tests.